clientv3/integration/kv_test.go

```diff
@@ -35,8 +35,8 @@ func TestKVPutError(t *testing.T) {
 	defer testutil.AfterTest(t)
 
 	var (
-		maxReqBytes = 1.5 * 1024 * 1024
-		quota       = int64(maxReqBytes * 1.2)
+		maxReqBytes = 1.5 * 1024 * 1024      // hard coded max in v3_server.go
+		quota       = int64(4 * 1024 * 1024) // the grpc max
```

Comment:
That was an attempt to resolve this error:

    -- FAIL: TestKVPutError (0.12s)
    kv_test.go:54: expected etcdserver: request is too large, got etcdserver: mvcc: database space exceeded

From what I could tell, ```database space exceeded``` means the quota limit was reached, so I increased the quota to the max, which is 4 MiB (limited by grpc max message size).